### ISSUE TITLE
Training records can leave empty sample

### ISSUE THREAD

**Issue by andrewvanbreda** [Indicia-Team/warehouse#228]
I noticed a problem when working on a project, am not sure how this will be resolved. The design of the system currently is that training records are set at the occurrence level, but this is theoretically an issue if we are interested in empty samples.
For a project test I created some occurrence training records, but the report was showing this a "proper" sample/visit which just didn't have any occurrences. This situation of not sightings was of interest to the project but was not correct in this scenario as it was only like that due to the way training records work.
Would it be better to have training set at a sample level as well?

**Comment by johnvanbreda** [comment]
Yes, or it might make sense to have the training flag at sample level instead as discussed the other day - though this would be more work.

**Comment by kazlauskis** [comment]
Hi, I also have this requirement where NPMS App surveys can have no occurrences and so no way to send an empty training sample. 

Also, I feel this makes me more sense to set a training on a parent level and treating its children models as training rather than the opposite, ie. I wouldn't consider sending one of the occurrences within a sample as training while other being valid models.
